components/ProfileView/QuestsToReview.tsx

```diff
@@ -86,8 +86,10 @@ const QuestChainStatusView: React.FC<{
   questChain: graphql.QuestChainReviewInfoFragment;
 }> = ({ questChain: chain }) => (
   <NextLink
-    as={`/chain/${chain.chainId}/${chain.address}/review`}
-    href={`/chain/[chainId]/[address]/review`}
+    as={`/${AVAILABLE_NETWORK_INFO[chain.chainId].urlName}/${
+      chain.address
+    }/review`}
+    href={`/[chainId]/[address]/review`}
```

Comment:
```suggestion
    href={`/[network]/[address]/review`}
```

Need to change this wherever applicable.